relay-profiling/src/measurements.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct Measurement {
+    // nanoseconds elapsed since the start of the profile (wall clock)
+    elapsed_since_start_ns: u64,
+    unit: String,
+    value: f64,
+}
```

Comment:
How is this planned to interact with built-in measurements? See
https://github.com/getsentry/relay/blob/d77c489292123e53831e10281bd310c6a85c63cc/relay-general/src/protocol/measurements.rs#L55

Reply:
I don't think it's necessarily planned to interact with the built-in one.

It's more internal to profile. 

Also, for us it's not a map `[String --> Measurement]` but rather a map `[String --> Vec<Measurement>]` (measurements tracked over time).

We also have a specific field  like `elapsed_since_start_ns`